docs/dg/dev/third-party-module-development/ensure-quality.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Ensure Quality
+description: Ensure Quality
+last_updated: Jun 7, 2024
+template: howto-guide-template
+
+---
+
+<style>
+  table th, table td {
+    white-space: nowrap;
+  }
+</style>
+
+**TBD: Commands must be updated after https://spryker.atlassian.net/browse/FRW-8436 is done (Check if we need it at all)  **
+
+## Tools
+
+To make sure that code quality matched the Spryker standards it is important to execute code quality tools, here is the list:
+
+| Command                                                                                 | Description                                               |
+|:----------------------------------------------------------------------------------------|:----------------------------------------------------------| 
+| console code:propel:validate-abstract -m YourCompanyName.YourModuleName                 | Checks Abstract classes for Propel tables exist in core   |
+| console propel:schema:validate                                                          | Validates Propel the schema files.                        |
+| console propel:schema:validate-xml-names                                                | Validates Propel XML element name rules for schema files. |
+| console transfer:validate                                                               | Validates transfer XML definition files.                  |
+| spryker-dev-console dev:validate-module-schemas YourCompanyName.YourModuleName          | Validates schema definitions for each module.             |
+| spryker-dev-console dev:validate-module-databuilders YourCompanyName.YourModuleName     | Validates databuilder XML files for each module.          |
+| spryker-dev-console dev:module-meta-files -d -v YourCompanyName.YourModuleName          | Updates meta files for each module.                       |
```

Comment:
I think it would be nice to explain also why in the description and if this can be skipped. This comment is for the whole column.

Reply:
> and if this can be skipped
These are mandatory commands to execute.